File: Greedy, BruteForce/Java/16968 차량 번호판 1.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      String s = sc.nextLine();
      int answer = (s.charAt(0) == 'd') ? 10 : 26;
      for(int i = 1 ; i < s.length(); i++) {
          if(s.charAt(i) == 'c') {
              if(s.charAt(i) == s.charAt(i-1)) answer *= 25;
              else answer *= 26;
          }
          else {
              if(s.charAt(i) == s.charAt(i-1)) answer *= 9;
              else answer *= 10;
          }
      }
      System.out.println(answer);
    }
}

File: Greedy, BruteForce/Kotlin/16968 차량 번호판 1.kt
import java.util.*
import kotlin.math.*

fun main(args: Array<String>) {
    var sc = Scanner(System.`in`)
    var s = sc.nextLine(); var answer = if(s[0] == 'd') 10 else 26
    for(i in 1 until s.length) {
        if(s[i] == 'd') {
            if(s[i] == s[i-1]) answer *= 9
            else answer *= 10
        }
        else {
            if(s[i] == s[i-1]) answer *= 25
            else answer *= 26
        }
    }
    println(answer)
}
